Publisher/nl/email-editor.md

```diff
@@ -5,12 +5,12 @@ In Marketing Suite kun je in de **[E-mail-editor](https://ms.copernica.com/nl#/t
 - Drag-and-drop-templates
 - Drag-and-drop-templates (oud)
 
-Met de Drag-and-drop-editor maak je snel en eenvoudig responsive e-mails door het slepen van elementen binnen je template. Wil je meer vrijheid in de vormegeving van je template kun je HTML-templates gebruiken.
+Met de Drag-and-drop-editor maak je snel en eenvoudig responsive e-mails door het slepen van elementen binnen je template. Met HTML-templates heb je meer vrijheid in de vormgeving omdat je hier niet vast zit aan standaard elementen.
```

Comment:
Drag-and-drop-editor vind ik wel verwarrend.
Is er een E-mail-editor en drag-and-drop-editor?

Ik zou zeggen: Met drag-and-drop-templates maak je snel en eenvoudig responsive e-mails door het slepen van elementen.

Reply:
Er is geen 'E-mail-editor' en 'Drag-and-drop-editor' omdat beide binnen dezelfde editor vallen (de E-mail-editor). Ik heb het aangepast naar de zin zoals je voor hebt gesteld.